code/modules/mob/mob_defines.dm

```diff
@@ -83,6 +83,7 @@
 	var/resting = 0			//Carbon
 	var/lying = 0
 	var/lying_prev = 0
+	var/is_shifted = FALSE // Vore Edit; pixel shifting
```

Comment:
This should have VOREStation Edit, mostly to make it easier to find edits via the search feature.